### DISCUSSION TITLE
如何在ArrayItems中获取删除节点的index

### DISCUSSION THREAD

**Discussion by 1owrie** [alibaba/formily#1752] [Q&A 2.x] [answered]
点击删除时在x-component-props中添加了点击事件deleteItem,如何在这个方法里获取删除节点的index呢？
sandbox: https://codesandbox.io/s/test-n31yz?file=/App.tsx:665-675

**Comment by liuweiGL** [comment, marked as answer]
https://codesandbox.io/s/test-forked-96ptp?file=/App.tsx

**Reply by 1owrie** [reply]
十分感谢
